src/test/java/com/github/jmchilton/blend4j/galaxy/ToolDataTest.java

```diff
@@ -0,0 +1,74 @@
+package com.github.jmchilton.blend4j.galaxy;
+
+import com.github.jmchilton.blend4j.galaxy.beans.TabularToolDataTable;
+import com.sun.jersey.api.client.ClientResponse;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ToolDataTest {
+    private GalaxyInstance instance;
+    private ToolDataClient client;
+
+    @BeforeMethod
+    public void init() {
+        instance = TestGalaxyInstance.get();
+        client = instance.getToolDataClient();
+    }
+
+    @Test
+    public void testGetDataTables() {
+        final List<TabularToolDataTable> toolDataTables = client.getDataTables();
+        assert toolDataTables != null;
+        assert ! toolDataTables.isEmpty();
+
+        for (final TabularToolDataTable toolDataTable : toolDataTables) {
+            assert toolDataTable.getName() != null;
+        }
+    }
+
+    @Test
+    public void testShowDataTable() {
+        final List<TabularToolDataTable> toolDataTables = client.getDataTables();
+        assert toolDataTables != null;
+        assert ! toolDataTables.isEmpty();
+
+        for (final TabularToolDataTable toolDataTable : toolDataTables) {
+            assert toolDataTable.getName() != null;
+            TabularToolDataTable dataTable = client.showDataTable(toolDataTable.getName());
+            assert dataTable != null;
+        }
+
+    }
+
+    @Test
+    public void testShowDataTableRequest() {
+        final List<TabularToolDataTable> toolDataTables = client.getDataTables();
+        assert toolDataTables != null;
+        assert ! toolDataTables.isEmpty();
+
+        for (final TabularToolDataTable toolDataTable : toolDataTables) {
+            assert toolDataTable.getName() != null;
+            ClientResponse clientResponse = client.showDataTableRequest(toolDataTable.getName());
+            assert clientResponse != null;
+        }
+
+    }
+
+    @Test
+    public void testTableContents() {
+        final TabularToolDataTable igvBroadGenomes = client.showDataTable("igv_broad_genomes");
+        assert igvBroadGenomes != null;
+
+        final ArrayList<String> headers = igvBroadGenomes.getColumns();
+        System.out.println(headers.get(0) + "\t" + headers.get(2));
```

Comment:
Instead of using `System.out.println`, could you print through a logger?  That is:

1. Add the following line to the top of the class

    ```java
    private static final Logger logger = LoggerFactory.getLogger(ToolDataTest.class);
    ```

2. Change `System.out.println()` to `logger.debug()`.

By default, debug messages won't be printed out, but they can be turned on by adjusting the log level in the file `src/test/resources/log4j.properties`.  For example by adding

```
log4j.logger.com.github.jmchilton.blend4j=DEBUG
```

Reply:
Also, instead of just printing out some of the fields, could you add assert statements to make sure some of them exist?